tests/spec/core/linter-rules/internal-slot-warn-spec.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+describe("Core Linter Rule - 'check-internal-slots'", () => {
+  const ruleName = "check-internal-slots";
+  const config = {
+    lint: { [ruleName]: true },
+  };
+  let rule;
+  beforeAll(async () => {
+    rule = await new Promise(resolve => {
+      require([`core/linter-rules/${ruleName}`], ({ rule }) => resolve(rule));
+    });
+  });
+  const doc = document.implementation.createHTMLDocument("test doc");
+  beforeEach(() => {
+    // Make sure every unordered test get an empty document
+    // See: https://github.com/w3c/respec/pull/1495
+    while (doc.body.firstChild) {
+      doc.body.removeChild(doc.body.firstChild);
+    }
+  });
+
+  it("return error when no '.' between var and a for internal slots", async () => {
```

Comment:
when writing "it()" functions always make sure the description reads like "it does X". So:

```suggestion
  it("only returns an error when there is no '.' between var and an internal slot", async () => {
```